components/script/dom/webglrenderingcontext.rs

```diff
@@ -644,6 +647,12 @@ impl WebGLRenderingContextMethods for WebGLRenderingContext {
             }
         } else {
             constants::NO_ERROR
+            //Todo(Mortimer): Enable GPU getError() when the webtraits repo is merged
+            //let (sender, receiver) = ipc::channel().unwrap();
+            //self.ipc_renderer
```

Comment:
Ok, this is not required in this PR.

I thought that was interesting to fallback to glGetError so devs can check unexpected errors in problematic GPUs or errors we don't handle yet. I'll check how the asserts are done in the renderer thread. Just curious about performance, are the checks done in release builds too or they should be enabled with a command line option?